src/libs/OptionsListUtils.js

```diff
@@ -186,7 +186,9 @@ function getSearchText(report, reportName, personalDetailList, isChatRoomOrPolic
     if (!isChatRoomOrPolicyExpenseChat) {
         for (let i = 0; i < personalDetailList.length; i++) {
             const personalDetail = personalDetailList[i];
-            searchTerms = searchTerms.concat([personalDetail.displayName, personalDetail.login.replace(/\./g, '')]);
+
+            // The regex below is used to remove dots only from the local part of the user email (local-part@domain)
+            searchTerms = searchTerms.concat([personalDetail.displayName, personalDetail.login, personalDetail.login.replace(/\.(?=[^\s@]*@)/g, '')]);
```

Comment:
Let's wait for @cead22 then

Reply:
@rushatgabhane This is more like a required thing to do, we can fix it but it will effect the performance.
Check my X Y analogy in the GH issue: https://github.com/Expensify/App/issues/12026#issuecomment-1329581613

We use X to search for Y
X must be constant
Y can be altered.

We can remove dots from Y because it's a straightforward task
But we cannot add dots to Y because we don't know where to add the dots exactly

We can fix this by changing X by removing the dots from X, but this will cause unwanted search queries

Reply:
>firstlast@domain.com matches firstlast@domain.com and first.last@domain.com
>first.last@domain.commatches only first.last@domain.com

This sounds good to me, if you're explicitly searching for a `.`, I think it's ok to expect only matches that have a `.`